### PR TITLE
Primitives should not be boxed just for Strign conversion

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

### CODE REVIEW

**Comment by jbonofre** [comment]
R: @jbonofre

File: webconsole/gogo/src/main/java/org/apache/karaf/webconsole/gogo/Terminal.java [hunk 1]
```diff
@@ -1710,14 +1710,14 @@ public synchronized String pipe(String d) {
                 vt100_keyfilter_escape = true;
             } else if (c == 127) {
                 if (vt100_mode_backspace) {
-                    o += (char) 8;
+                    o += Integer.toString(8);
```
Comment (jbonofre) [inline webconsole/gogo/src/main/java/org/apache/karaf/webconsole/gogo/Terminal.java]: It's a `String` whereas we cast to `char`.

Comment (apupier) [reply webconsole/gogo/src/main/java/org/apache/karaf/webconsole/gogo/Terminal.java]: yes but the char is then autocasted to String. I think that we can get rid of this step, isn't it?

Comment (jbonofre) [reply webconsole/gogo/src/main/java/org/apache/karaf/webconsole/gogo/Terminal.java]: Agree, it's just a personal note for the merge ;)

**Pull request closed without merging**